app/javascript/mastodon/features/notifications/components/grant_permission_button.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import Icon from 'mastodon/components/icon';
+
+export default class GrantPermissionButton extends React.PureComponent {
+
+  static propTypes = {
+    onClick: PropTypes.func.isRequired,
+  };
+
+  render () {
+    return (
+      <button className='text-btn column-header__setting-btn' tabIndex='0' onClick={this.props.onClick}>
+        <Icon id='sliders' /> <FormattedMessage id='notifications.grant_permission' defaultMessage='Grant permission' />
```

Comment:
Why is the icon `sliders`? In the banner, I used the icon to point to the column settings area, but it is not the icon for permissions or desktop notifications.

Reply:
This seems to be the most fitting “settings icon” to me, although it's already used for column settings, so I understand it may be confusing. I don't know which icon would be better.